src/pdn/BUILD

```diff
@@ -60,6 +60,7 @@ cc_library(
         "//src/utl",
         "@boost.geometry",
         "@boost.polygon",
+        "@boost.stacktrace",
```

Comment:
This dependency comes from the swig file, should it be expressed tcl_wrap_cc instead? (This is true for all tools)